### PR TITLE
SOAR-2131 Add docs_url to pluginspec

### PR DESCRIPTION
**Pull request by ekim-r7**
### Description

Describe the proposed changes:

  - Add docs_url to pluginspec

### Testing
```
[*] Validating plugin at .

[*] Running Integration Validators...
[*] Executing validator HelpValidator
[*] Executing validator ChangelogValidator
[*] Executing validator RequiredKeysValidator
[*] Executing validator UseCaseValidator
[*] Executing validator SpecPropertiesValidator
[*] Executing validator SpecVersionValidator
[*] Executing validator FilesValidator
[*] Executing validator TagValidator
[*] Executing validator DescriptionValidator
[*] Executing validator TitleValidator
[*] Executing validator VendorValidator
[*] Executing validator DefaultValueValidator
[*] Executing validator IconValidator
[*] Executing validator RequiredValidator
[*] Executing validator VersionValidator
[*] Executing validator DockerfileParentValidator
[*] Executing validator LoggingValidator
[*] Executing validator ProfanityValidator
[*] Executing validator AcronymValidator
[*] Executing validator JSONValidator
[*] Executing validator OutputValidator
[*] Executing validator RegenerationValidator
[*] Executing validator HelpInputOutputValidator
[*] Executing validator SupportValidator
[*] Executing validator RuntimeValidator
[*] Plugin successfully validated!

----
[*] Total time elapsed: 927.44ms
```

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

File: microsoft_atp/help.md [hunk 1]
```diff
@@ -19,12 +19,6 @@ This plugin utilizes the [Microsoft ATP API](https://docs.microsoft.com/en-us/wi
 
 ## Setup
 
-This plugin uses the Windows Defender ATP API. It will use an Azure application to connect to the API and run
```
Comment (jschipp-r7) [inline microsoft_atp/help.md:22]: Add this paragraph back - the tooling removed it - it's a bug so will have to revert or commit it back again.

File: microsoft_atp/help.md [hunk 2]
```diff
@@ -174,10 +168,7 @@ Example input:
   "expiration_time": "2020-12-12T00:00:00Z",
   "indicator": "220e7d15b011d7fac48f2bd61114db1022197f7f",
   "indicator_state": true,
-  "rbac_group_names": [
-    "group1",
-    "group2"
-  ],
+  "rbac_group_names": "[\"group1\",\"group2\"]",
```
Comment (jschipp-r7) [inline microsoft_atp/help.md:171]: Revert back to the list view too

File: microsoft_atp/help.md [hunk 3]
```diff
@@ -760,6 +751,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 4.5.1 - Add docs_url to plugin spec
```
Comment (jschipp-r7) [inline microsoft_atp/help.md:754]:
```suggestion
* 4.5.1 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://docs.rapid7.com/insightconnect/microsoft-defender-ATP)

```

**Review by jschipp-r7 — CHANGES_REQUESTED** [review]

File: microsoft_atp/help.md [hunk 4]
```diff
@@ -751,7 +751,8 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
-* 4.5.1 - Add docs_url to plugin spec
+* 4.5.1 - Add `docs_url` to plugin spec with link to [plugin setup guide](https://docs.rapid7.com/insightconnect/microsoft-defender-ATP)
+
```
Comment (jschipp-r7) [inline microsoft_atp/help.md:763]: You have an extra new line here, that breaks the formatting

**Review by mhofert-r7 — COMMENTED** [review]
Per Jon add the comment back then you are good to go

**Review by mhofert-r7 — APPROVED** [review]

**Review by ncramer-r7 — APPROVED** [review]

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**